gulp/tasks/test.js

```diff
@@ -2,11 +2,13 @@
 
 var gulp = require( 'gulp' );
 var $ = require( 'gulp-load-plugins' )();
-var protractor = require( 'gulp-angular-protractor' );
-var exec = require( 'child_process' ).exec;
+var childProcess = require( 'child_process' );
+var exec = childProcess.exec;
```

Comment:
If we're going to define this one, we should define `spawn` as well. Otherwise just use `childProcess.exec()` on L:37 and L:47